### ISSUE TITLE
[Firefox] Popup in high-contrast theme

### ISSUE THREAD

**Issue by vegan4all** [uBlock-LLC/uBlock#1242]
Hi,

Firefox 32.0a1 (but same issue with later versions, and waterfox)

tools>options>content>colors
deselect  "Use system colors"
deselect  "Allow pages to choose their own colors, instead of my selections above"

This really degrades the usability of the toolbar popup
The worst consequence is that the colors of the green/grey/red boxes all vanish, so that you can not see which rules/sites are allowed, nooped or blocked.
Surely there would be some way to code the popup to make it more compatible with custom colors
(policeman popup is unaffected by custom colors)

I tried changing a background-color to a background-image,
as in here:
# gotoPrefs {

```
display: block;
margin: 0;
border: 0;
padding: 4px;
padding-top: 5px;
color: #eee;
font-weight: bold;
background-image: url(file:///C:/Winter.jpg) !important;
```

and the top bar changes to part of this picture that i placed on the root of the c: drive
But when i use custom colors it reverts to black (or whatever custom background color was selected)

I tried comparing the popup.css code with the policeman popup.css code to see why one popup changes with custom colors, whilst the other remain unaffected - but they are too dissimilar for me to work out.
Tried inserting [background-color: -moz-field;] in various places, to no avail.

Obviously, i'm no expert in css

Wouldn't be surprised if it was an easy fix.
Not everyone likes using default colors for viewing webpages

Regards

![popup-no-custom-colors](https://cloud.githubusercontent.com/assets/11906335/7104249/8a83e624-e115-11e4-8de5-920508b5851f.JPG)
![popup-custom-colors](https://cloud.githubusercontent.com/assets/11906335/7104250/942dc2c6-e115-11e4-809a-1c12e1c2711d.JPG)

**Comment by chrisaljoudi** [comment]
@vegan4all fix in f961cad, which makes it distinguishable what the state of the dynamic filter table is.

Note: a hack to "force" Firefox to show colors in a mode that explicitly disallows them would be suboptimal, so I don't plan on doing that.

**Comment by tophf** [comment]
This *must\* be an option. The added text in f961cad creates a lot of info noise and many users (who knows, maybe most?) of the advanced mode don't need it.
